src/callbacks/check_gradients.cpp

```diff
@@ -133,10 +133,13 @@ struct CheckWeightsFunctor : DefaultErrorReporter
     // Get weights matrix and gradient
     const auto& weights_matrix = dtw.get_values();
     const auto& gradient = dtw.get_optimizer()->get_gradient();
-
+    // std::cout << "*** CHECKING WEIGHTS: NAME=" << dtw.get_name() << ", SIZE="
```

Comment:
Eventually this debug code should be removed.

Reply:
Done